conf/machine/include/imx-base.inc

```diff
@@ -164,6 +164,7 @@ DEFAULTTUNE:mx8m-generic-bsp   ?= "cortexa53-crypto"
 DEFAULTTUNE:mx8qm-generic-bsp  ?= "cortexa72-cortexa53-crypto"
 DEFAULTTUNE:mx8x-generic-bsp   ?= "cortexa35-crypto"
 DEFAULTTUNE:mx8ulp-generic-bsp ?= "cortexa35-crypto"
+DEFAULTTUNE:mx9-generic-bsp    ?= "cortexa35-crypto"
```

Comment:
Not sure this tune is correct here.

IIRC, `imx93` has _A55_ and it is closer to _A53_ than to _A35_.

There is no `DEFAULTTUNE` for _A55_ available in _OE-Core_, but in this case I would rather set the tune to `cortexa53-crypto` instead.

Reply:
Looks like A55 is available?

https://github.com/openembedded/openembedded-core/blob/master/meta/conf/machine/include/arm/armv8-2a/tune-cortexa55.inc

Reply:
True, thanks for the hint!

Sorry, I was searching in the different folder (`armv8a`) and did not realize that _OE-Core_ had a separate folder laid out for _ARMv8.2-A_ architecture.

Then it's indeed not correct - the tune shall be set to point to the correct one.

Reply:
changed

Reply:
@angolini 

There is care to be taken here, as changing `DEFAULTTUNE` to plain `cortexa55` here would not imply that Advanced SIMD instructions would be enabled as well... This was enabled implicitly by `-crypto` suffix for `imx8m` series, but as far as I can see this suffix is not (yet) supported in _OE_Core_ for _A55_ profile.

I'm not sure if it's a miss in _OE-Core_ or it is not brought up deliberately, but without it - performance might suffer as ASIMD instructions would not be emitted by compiler.

@thochstein @otavio I'm a bit torn here: from one side - the new tune matches to what SOC has in the Cortex-A complex, but on the other hand - resulting compiled binaries might suffer performance or size difference if it is enabled like this.

Shall it be taken as it is presented in this PR, or perhaps first `-crypto` suffix shall be submitted to _OE-Core_?

I'm not sure: if `imx93` implements full _A55_ profile(@thochstein ❔) - then `-crypto` suffix would really be desired; if not - then this change is good to go.

-- andrey

Reply:
I will have to ask. We have been setting `DEFAULTTUNE:mx9-nxp-bsp    ?= "armv8a-crc-crypto"` in our distros, and I see now that we neglected to add an optimized tuning from the BSP layer.

Reply:
I'd add it for our tune, so it uses by default @zandrey.

Reply:
so far, I will let it be. Waiting to see if @thochstein brings some news

Reply:
@thochstein

Good catch! Yes, indeed it is!

It is actually coming from the top include, and shall be enabled for `cortexa55`.

I would suggest to set `DEFAULTTUNE` straight to `cortexa55` here.

Reply:
@zandrey, great! I've also received confirmation that i.MX 93 implements full A55 profile.

Reply:
great! So I'm resolving this as we all agreed (and my current version is already fixed)